thunder/core/jit_ext.py

```diff
@@ -829,6 +829,51 @@ def _general_jit_bool_lookaside(wrapped_x: Any) -> bool | INTERPRETER_SIGNALS:
 
 _general_jit_lookaside_map[bool] = _general_jit_bool_lookaside
 
+
+def _torch_module_named_member_lookaside(model, get_member_func, prefix="", recurse=True, remove_duplicate=True):
+    memo = set()
+    modules = model.named_modules(prefix=prefix, remove_duplicate=remove_duplicate) if recurse else [(prefix, model)]
+    for module_prefix, module in modules:
+        members = get_member_func(module)
+        for k, v in members.items():
+            # We use name of the proxy as id.
+            id_v = v.name
+            if id_v is None or id_v in memo:
+                continue
+            if remove_duplicate:
+                memo.add(id_v)
```

Comment:
The problem with memo is why my idea was to run `named_parameters` inside the lookaside rather than the the impl and only keep a list of names (but a deduplicated one!). Then, in an `_impl` with `_interpret_call` we can take the list of names use `m.get_parameter(n))` to get the deduped list. Or we could just make set of names in the lookaside and just return those that are in the set here (that is likely quicker).
WDYT?